packages/@lwc/engine-dom/src/apis/build-custom-element-constructor.ts

```diff
@@ -5,14 +5,11 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
  */
 
-import { create, isUndefined, keys } from '@lwc/shared';
 import {
     createVM,
     connectRootElement,
     disconnectRootElement,
-    getAttrNameFromPropName,
```

Comment:
however this is still used in other places... can we get rid of it as well in the future? @pmdartus

Reply:
`getAttrNameFromPropName` is also used in: 
- the [`@engine/dom` renderer](https://github.com/salesforce/lwc/blob/master/packages/@lwc/engine-dom/src/renderer.ts#L125) - we can certainly move this one to the compiler as the TODO states
- the [`@engine/server` renderer](https://github.com/salesforce/lwc/blob/master/packages/@lwc/engine-server/src/renderer.ts#L120-L184) - this one will be a little more complex with the current architecture. This is something that we will certainly be able to get rid of if we go down the pass of a VDOM-less approach in the engine.

IMO the `getAttrNameFromPropName` has nothing to do the engine code. It's certainly something we should move to `@lwc/shared`. What do you think?

Reply:
This should be done in another PR, issue created: https://github.com/salesforce/lwc/issues/2091